### PR TITLE
Modify GetProperties overload

### PR DESCRIPTION
**Pull request by jbagga**
Addresses: https://github.com/aspnet/Common/issues/219
See https://github.com/aspnet/Common/pull/228

cc @rynowak

### CODE REVIEW

File: src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/HtmlHelper.cs [hunk 1]
```diff
@@ -203,7 +204,7 @@ public ITempDataDictionary TempData
 
             if (htmlAttributes != null)
             {
-                foreach (var helper in HtmlAttributePropertyHelper.GetProperties(htmlAttributes))
+                foreach (var helper in HtmlAttributePropertyHelper.GetProperties(htmlAttributes.GetType().GetTypeInfo()))
```
Comment (pranavkm) [inline src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/HtmlHelper.cs]: Shouldn't `GetType()` suffice?

Comment (jbagga) [inline src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/HtmlHelper.cs]: Not if I make this change https://github.com/aspnet/Mvc/pull/6276/files#diff-bde884ef326029a4210b64620a71026fR16 Should I change the parameter to `Type type` instead?

Comment (pranavkm) [inline src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/HtmlHelper.cs]: Sure. As long as it works, that should suffice

**Comment by jbagga** [comment]
@pranavkm Updated

**Pull request merged**